### ISSUE TITLE
Version is a required field in `info`

### ISSUE THREAD

**Issue by dbanty** [poem-web/poem#87]
## Expected Behavior

`"version"` is required by the library or filled in automatically in order to comply with [this part of the spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#info-object).

Perhaps `OpenApiService::new` could required this and `title` (also required) as parameters? Though that'd be a breaking change which you may want to avoid. You could also make the builder return a `Result` to check for required params at startup, but that is also a breaking change.

The examples should also probably include version, perhaps using a macro to include the `Cargo.toml` version (I think this is possible and is probably what I'll do eventually).

## Actual Behavior

`"version"` can be omitted (as it is in the examples I followed) which can lead to validation errors in other tools.

## Steps to Reproduce the Problem

Omit the `.version()` method in setting up `OpenApiService` and observe that the generated document does not contain a `version` field under `info`.

## Specifications

```toml
[dependencies]
poem = { version = "1.0.24", features = ["multipart", "tempfile"] }
poem-openapi = "1.0.20"
```

**Comment by sunli829** [comment]
I will add the `version` and `title` parameters to `OpenApiService::new`, although this is a breaking change. 🙂

**Comment by sunli829** [comment]
Fixed in `v1.0.27`. 😁
